### PR TITLE
Improve accessibility by announcing changed values

### PR DESCRIPTION
**Pull request by marcuswu**
This change announces the changed value. I do it in the animation check because it's not necessary to announce the value change during initialization. If you want to suggest a better way to do this, I'm open to ideas.

### CODE REVIEW

**Comment by cocoatoucher** [comment]
Hey sorry for the delay, I think it looks good. Do you mind bumping a new version for the pod spec as well?

**Comment by marcuswu** [comment]
Sure, just pushed up a version bump

File: Source/AIFlatSwitch.swift [hunk 1]
```diff
@@ -176,6 +176,7 @@ import UIKit
 		// Animate to new state
 		if animated {
 			self.addAnimations(desiredSelectedState: isSelectedInternal)
+            self.accessibilityValue = isSelected ? "checked" : "unchecked"
```
Comment (cocoatoucher) [inline Source/AIFlatSwitch.swift]: @marcuswu I also noticed indentation looks weird here, is it same for you?

Comment (marcuswu) [reply Source/AIFlatSwitch.swift]: Fixed -- I had spaces vs the rest of the file which uses tabs. It appeared the same in my editor.

**Comment by marcuswu** [comment]
Does it look good? I'm happy to make further changes if necessary.

**Comment by cocoatoucher** [comment]
Hey @marcuswu sorry again for the delay 😔 and thanks for your contribution
Also pushed 1.0.3 to cocoapods (https://cocoapods.org/pods/AIFlatSwitch)

**Comment by marcuswu** [comment]
Awesome, thanks!

**Pull request merged**